templates/srTakeover_hook.j2

```diff
@@ -0,0 +1,103 @@
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes %}
+{% if node.host == host and node.primary is defined %}
```

Comment:
Do we need to copy this file in the primary node?

Using this comparison you are only considering this template for the primary. If this must be used in the other machine (the machine we need to set as cost optimized) we will need to add an extra parameter to know where we have to create this file, and get the primary data too (maybe we need to loop twice, to get the cost optimized node and the primary)

Reply:
Hi @arbulu89 This file need to be copied only to secondary node, and we need the user_name ('SYSTEM')which I could only find under the primary/userkey info, and was not under secondary info. 
Maybe in pillar data, we can add user_name on secondary, along with the extra parameter to distinguish for cost-optimized. What do you think?

Reply:
I think we don't need to add any extra user_name parameter. I would do something similar that we do in: https://github.com/SUSE/saphanabootstrap-formula/blob/master/hana/copy_ssfs.sls

First loop through the nodes to find the node where we have to create this file (looking for scenario-type: cost-optimized entry in pillar data for example). And then loop again to find the primary that matches with the remote host of the secondary. We can get all the data with that.